pvlib/atmosphere.py

```diff
@@ -374,10 +374,13 @@ def first_solar_spectral_correction(pw, airmass_absolute, module_type=None,
             * 'cdte' - First Solar Series 4-2 CdTe modules.
             * 'monosi', 'xsi' - First Solar TetraSun modules.
             * 'multisi', 'polysi' - multi-crystalline silicon modules.
+            * 'cigs' - anonymous copper indium gallium selenide PV module
+	    * 'asi' - anonymous amorphous silicon PV module
```

Comment:
Hi Will. Sorry for the long delay. I tried making the recommended changes. Please confirm whether they were made successful and if you need me to make any additional changes.